oauth2client/contrib/django_util/__init__.py

```diff
@@ -210,6 +270,23 @@ def _get_oauth2_client_id_and_secret(settings_instance):
                 "GOOGLE_OAUTH2_CLIENT_SECRET in settings.py")
 
 
+def _get_storage_model():
+    """This configures whether the credentials will be stored in the session
```

Comment:
If the audience of this comment is users of this function, then a doc string is appropriate and it should be [formatted properly with a single-line summary string and a Returns: section](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments). 

If the audience of this comment is future maintainers of this function, this text should be an octothorpe-delimited implementation comment.